tests/org.eclipse.ui.workbench.texteditor.tests/META-INF/MANIFEST.MF

```diff
@@ -9,15 +9,18 @@ Export-Package:
  org.eclipse.ui.workbench.texteditor.tests,
  org.eclipse.ui.workbench.texteditor.tests.minimap,
  org.eclipse.ui.workbench.texteditor.tests.revisions,
- org.eclipse.ui.workbench.texteditor.tests.rulers
+ org.eclipse.ui.workbench.texteditor.tests.rulers,
+ org.eclipse.ui.findandreplace
 Require-Bundle: 
  org.eclipse.core.runtime;bundle-version="[3.29.0,4.0.0)",
  org.eclipse.jface.text;bundle-version="[3.5.0,4.0.0)",
  org.eclipse.ui.workbench.texteditor;bundle-version="[3.5.0,4.0.0)",
  org.eclipse.ui;bundle-version="[3.5.0,4.0.0)",
  org.junit;bundle-version="4.12.0",
  org.eclipse.text.tests;bundle-version="[3.5.0,4.0.0)",
- org.eclipse.core.expressions;bundle-version="[3.5.0,4.0.0)"
+ org.eclipse.core.expressions;bundle-version="[3.5.0,4.0.0)",
+ org.mockito.mockito-core
```

Comment:
It is sufficient to have package imports. The additional bundle dependency for mockito is not necessary.